packages/gestalt/src/Masonry/MeasureItems.js

```diff
@@ -1,81 +0,0 @@
-// @flow strict
-import { Fragment, type Node as ReactNode, useLayoutEffect, useMemo } from 'react';
-import { type Cache } from './Cache';
-import { type Position } from './types';
-
-const layoutNumberToCssDimension = (n: number) => (n !== Infinity ? n : undefined);
-
-type Props<T> = {
-  baseIndex: number,
-  getPositions: (items: $ReadOnlyArray<T>) => $ReadOnlyArray<Position>,
-  items: $ReadOnlyArray<T>,
-  measurementStore: Cache<T, number>,
-  renderItem: ({
-    +data: T,
-    +itemIdx: number,
-    +isMeasuring: boolean,
-  }) => ReactNode,
-};
-
-export default function MeasureItems<T>({
```

Comment:
Isn't this the version of item measuring we want to keep using?

Reply:
can you elaborate? this code is only being hit if `__batchPaints` is passed in

Reply:
Right — isn't batching paints the behavior we want to keep?

Reply:
i don't think so (correct me if i'm wrong). `__batchPaints` will mean that all measurements are always grouped together vs just batching the paints for the pages that contain multi column items.

Currently, we have it configured to increase the batch size if the current list of items to measure contains a two column item: https://github.com/pinterest/gestalt/blob/master/packages/gestalt/src/Masonry.js#L597. i suspect we'll need to tune this as we increase support to N columns but we likely don't need to always batch paints

Reply:
Ah yeah I think you are correct. A bummer to lose a little bit of code modularization with the MeasureItems component but not a big deal — we'll do a lot more of that cleanup in the future.